### ISSUE TITLE
Opt Out

### ISSUE THREAD

**Issue by boehs** [AnswerDotAI/llms-txt#3]
If llms.txt is used to supplement perceived failures of robots.txt, it should also correct in the other direction. I'd like a way to specify a

1. Blanket opt-out of my site being used by LLMs
2. Or a price tag for the usage of my site's content, alongside instructions on how to license it.

**Comment by shayneoneill** [comment]
Yeah seconding this. Fences make good neighbors, and ward off lawsuits.

While I do think that in some respect this is a functionality covered by robots.txt, it would appear much of the AI industry seems to think robots.txt doesnt apply to them, so a more explicit llms.txt set of permissions clause, 

Something like.
```
Permisssions
##########
Precedence:  trainonly, referenceonly, allow, disallow
disallow: / 
trainonly: /blog/archives
referenceonly: /current-data
allow: /blog
```

What this is saying is:  the precedence of most valid to least valid is <blah> 
So what its saying is, "The default here is say away. However you may train your data on the archives but not reference it in answers, you may only reference current-data, but not train on it. For the /blog directory however, you may do both, but since trainonly has higher preferences you must exclude the archives from referencing.

This would provide a  way for websites to choose what can be referenced, what can be trained on, and what must be excluded.

**Comment by virtadpt** [comment]
This is of interest to us as well.
